src/schema/fair.ts

```diff
@@ -291,12 +291,26 @@ const FairType = new GraphQLObjectType({
               description: "Letter exhibitors group belongs to",
             },
             exhibitors: {
-              type: new GraphQLList(GraphQLString),
-              description: "Exhibitors sorted by name",
-            },
-            profile_ids: {
-              type: new GraphQLList(GraphQLString),
-              description: "Partner/Exhibitor default profile id",
+              description: "The exhibitor data.",
+              type: new GraphQLList(
+                new GraphQLObjectType({
+                  name: "FairExhibitor",
+                  fields: {
+                    exhibitor_name: {
```

Comment:
From a best practice / convention perspective, these fields should be named `id` and `name`

Reply:
Agreed.